### DISCUSSION TITLE
disable AIO update check and notification

### DISCUSSION THREAD

**Discussion by zzz929625** [nextcloud/all-in-one#1502] [Ideas] [answered]
First of all,thanks for nicelly work on this project.
I'm using AIO for my files and it's amazing.
Recently I have recive a lot update notifications,I want have a stable version and happy to use it without upgrade.
So is there a switch or config can disable update check or disable update notification?

**Comment by szaimen** [comment]
Hi, see https://github.com/nextcloud/all-in-one/discussions/1487#discussioncomment-4350012

**Comment by mikebakke** [comment]
The volume of notifications is manageable but what is not is the fact that a major update will break installed applications. I was running v24 and had to stop/start my master to do some system maintenance. On restart, I was told an update (to v25) was available, but I had no ability to decline the update and just restart at v24.

The issue is that after restart I found at least 5 applications which the system disabled as not being compatible. This is not a good approach as it removes control from the user over the system. I was keen to get up to 25 but did not gain enough in features to outweigh the losses.

No doubt the apps will gradually get fixed but until them I am stuck. 

Please - consider an approach where before an update is triggered, the user is informed which apps will be disabled and simply offer the choice to "freeze" the system in it's current state until released by the user.

**Comment by zzz929625** [comment]
When I upgrade to v25,find some applications broken,and I can't find app store from web anymore.So I can't upgrade applications from web.I don't know this happens to only me or I do it wrong.

**Reply by szaimen** [reply]
This has nothing to do with this thread.

**Comment by szaimen** [comment, marked as answer]
https://github.com/nextcloud/all-in-one/discussions/1487#discussioncomment-5225016
